wifiphisher/common/interfaces.py

```diff
@@ -243,6 +243,36 @@ def up_ifaces(self, ifaces):
         for i in ifaces:
             card = pyw.getcard(i.get_name())
             pyw.up(card)
+    
+    def _generate_random_address(self):
```

Comment:
@sorphron, how about if user doesn't specify any MAC address related options, should we randomized it too?